twistlock/tests/test_twistlock.py

```diff
@@ -39,6 +30,7 @@
     'twistlock.containers.compliance.count',
 ]
 
+customtag = "custom:tag"
```

Comment:
```suggestion
```

Reply:
There is one test that actually calls that variable (line 57). I can remove the variable and just hard code the tag in that test. What do you think?